packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/editInfoUtils/applyChange.ts

```diff
@@ -55,8 +55,8 @@ export default function applyChange(
         newSrc = event.newSrc;
     }
 
-    if (newSrc == editInfo.src) {
-        // If newSrc is the same with original one, it means there is only size change, but no rotation, no cropping,
+    if (newSrc == editInfo.src && !editInfo.wasResized) {
```

Comment:
Even if resized, you can still delete edit info. Because image will have width/height value. It can help us regenerate wasResized